src/components/VAudioDetails/VAudioDetails.vue

```diff
@@ -26,8 +26,8 @@
           <div v-if="audio.audio_set">
             <dt>{{ $t('audio-details.table.album') }}</dt>
             <dd>
-              <VLink :href="audio.audio_set.url">{{
-                audio.audio_set.name
+              <VLink :href="audio.audio_set.foreign_landing_url">{{
+                audio.audio_set.title
```

Comment:
Can you also update the `audio_set` type to have `title` and `url` properties (or even all the properties that are available from the catalog)?

https://github.com/WordPress/openverse-frontend/blob/698e608c28955f111b36ce20bd3134902a41d454/src/models/media.ts#L57

Because we haven't added this file to the tsconfig.json TS check isn't failing here when it should because `audio_set` is just a `string`.